src/pages/questionPage.js

```diff
@@ -1,72 +1,79 @@
-'use strict';
+"use strict";
 
-import { ANSWERS_LIST_ID } from '../constants.js';
-import { NEXT_QUESTION_BUTTON_ID } from '../constants.js';
-import { getQuestionElement } from '../views/questionView.js';
-import { createAnswerElement } from '../views/answerView.js';
-import { createCheatButton, showCorrectAnswerElement } from '../views/cheatAnswerView.js';
-import { quizData } from '../data.js';
-import { router } from '../router.js';
+import { ANSWERS_LIST_ID } from "../constants.js";
+import { NEXT_QUESTION_BUTTON_ID } from "../constants.js";
+import { getQuestionElement } from "../views/questionView.js";
+import { createAnswerElement } from "../views/answerView.js";
+import {
+  createCheatButton,
+  showCorrectAnswerElement,
+} from "../views/cheatAnswerView.js";
+import { quizData } from "../data.js";
+import { router } from "../router.js";
 
 export const initQuestionPage = (userInterface) => {
-const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
-const questionElement = getQuestionElement(currentQuestion.text);
-userInterface.appendChild(questionElement);
-const answersListElement = document.getElementById(ANSWERS_LIST_ID);
-for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
-  const answerElement = createAnswerElement(key, answerText);
-  answersListElement.appendChild(answerElement);
-  answerElement.addEventListener('click',getAnswer)
-}
-  
-// get next question handler
-document
+  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+  const questionElement = getQuestionElement(currentQuestion.text);
+  userInterface.appendChild(questionElement);
+  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+  for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
+    const answerElement = createAnswerElement(key, answerText);
+    if (key === currentQuestion.correct) {
+      answerElement.setAttribute("data-correct", "correct");
```

Comment:
I was thinking yesterday in the bed that maybe it's not such a good idea to print in the html which is the correct answer :) 

I thought a better approach might be to just add to each of the answers a data attribute with the letter of the answer (e.g. data-answer="b") when building the html. In this way you could use, in showCorrect function,  
```
const correctAnswer = quiz.currentQuestion.correctAnswer;
document.querySelector(`li[data-answer=${correctAnswer}]`);
```
or similar.

What do you think?